demos/python/dmps/with_schedules/demo_dmp_with_schedules_parameterizable.py

```diff
@@ -0,0 +1,87 @@
+# This file is part of DmpBbo, a set of libraries and programs for the
+# black-box optimization of dynamical movement primitives.
+# Copyright (C) 2018 Freek Stulp
+#
+# DmpBbo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# DmpBbo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with DmpBbo.  If not, see <http://www.gnu.org/licenses/>.
+"""Script for dmp_parameterizable demo."""
```

Comment:
I do not find in the file or the README.md what are the differences between the other demo and this parameterizable demo

Reply:
Initially, the demos were meant to be self-explanatory. Now they have grown and it is no longer the case. This is a larger issue for which I have openened #74. Thanks for pointing this out.